py3status/modules/dropboxd_status.py

```diff
@@ -86,14 +86,16 @@ def dropbox(self):
 
         if status == "Dropbox isn't running!":
             color = self.py3.COLOR_BAD
-            status = self.status_off
+            _format = self.status_off
         elif status == "Up to date":
             color = self.py3.COLOR_GOOD
-            status = self.status_on
+            _format = self.status_on
         else:
             color = self.py3.COLOR_DEGRADED
             if self.status_busy is not None:
-                status = self.status_busy
+                _format = self.status_busy
+
+        status = self.py3.safe_format(_format, {'status': status})
```

Comment:
I killed your update. The `newformat-dropbox` branch removes duplicate `{status}` so I think that's definitely the cleanest way to go... by getting rid of states, statuses, etc... It mean less things to document too. We let users tweak whatever they want in the `format`.

If you look again and agree, the action to eventually take, I think, is to fix TRUE/FALSE thing inside the formatter... so we can omit at least this.
```
TRUE = ''
FALSE = ' '

        is_busy = TRUE if is_busy else FALSE
        is_started = TRUE if is_started else FALSE
        is_updated= TRUE if is_updated else FALSE
```
I went with `''` / `' '`. If you want `'True'` / `'False'`, I can do it too.

I revisit and think `True`/`False` will look clean on the return (for debugging purpose) but won't be useful for users if they tried `{shuffle}` to get `True`. Still clean though. 

Will look, fix, and test this module right now.